static/js/spectators.js

```diff
@@ -28,5 +35,12 @@ export function login_to_access() {
             login_link,
         }),
     );
+
     $("#login_to_access_modal").modal("show");
+    is_login_to_access_open = true;
+
+    $("#login_to_access_modal").bind("hide", () => {
+        browser_history.return_to_web_public_hash();
+        is_login_to_access_open = false;
+    });
```

Comment:
I think we want to convert `login_to_access` to Micromodal; @ganpa3 can you help advice on doing that (or maybe just do it in a separate PR that we can rebase this commit on top of?)

Reply:
@ganpa3 thoughts? I haven't looked at the library, but can I do the port if it is blocker for this PR.

Reply:
#20381